tests/setup.js

```diff
@@ -82,12 +82,30 @@ const setUp = async () => {
   }));
 
   // Make sure all responses have 'Vary: X-Test-Run' to avoid conflicts between test runs.
-  config.push(api.post(`version/${version}/header`).form({
-    name: 'vary-test-run',
-    type: 'cache',
-    action: 'append',
-    dst: 'http.Vary',
-    src: '"X-Test-Run"',
+  config.push(api.post(`version/${version}/snippet`).form({
+    name: 'Vary X-Test-Run',
+    dynamic: 0,
+    type: 'fetch',
+    content: `
+if (!beresp.http.Vary) {
+  set beresp.http.Vary = "X-Test-Run";
+} else {
+  set beresp.http.Vary = beresp.http.Vary ", X-Test-Run";
```

Comment:
We hit https://github.com/elifesciences/fastly-iiif/pull/1#discussion_r216612833.

Reply:
Might work to append `, X-Test-Run`? Simplifies the regex to remove too?